_rules/autocomplete-valid-73f2c2.md

```diff
@@ -19,7 +19,7 @@ authors:
 
 ## Applicability
 
-The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is not empty (""), except if one of the following is true:
+The rule applies to any HTML `input`, `select` and `textarea` element with an `autocomplete` attribute that is not empty ("") or that is not only [whitespace](#whitespace), except if one of the following is true:
```

Comment:
This should be "not empty or only whitespace", or (better) "neither empty nor only whitespace".

Currently, the alternative is "(not empty) OR (not whitespace)" and `""` being (not whitespace) pass this applicability.

*i.e.* you wrote `applicability = (!(autocomplete=="")) || (!(autocomplete is only whitespace))` instead of `applicability = !( (autocomplete=="") || (autocomplete is only whitespace) )` 
(at least, that's how I understand the sentence)